pkg/ebpf/c/tracee.bpf.c

```diff
@@ -786,6 +797,7 @@ u64 last_module_insert_time = 0;
 bool hidden_old_mod_scan_done = false;
 static const int HID_MOD_RACE_CONDITION = -1;
 static const int HID_MOD_UNCOMPLETED_ITERATIONS = -2;
+static const int HID_MOD_COMPLETED_ITERATIONS = 0;
```

Comment:
I don't see this variable is being assigned any value different than 0, why not make it a constant then?

Reply:
It is a constant, isn't it?